apps/portlet-documents/src/main/webapp/js/attachmentService.js

```diff
@@ -189,24 +189,23 @@ export function uploadAttachment(workspaceName, driveName, currentFolder, curren
     });
 }
 
-export function linkUploadedAttachmentsToEntity(entityId, entityType, attachmentIds) {
-  attachmentIds.forEach(attachmentId => {
-    if (!attachmentId) {
-      throw new Error('Attachment Id can\'t be empty');
-    }
-  });
-  let params = {};
-  if (attachmentIds) {
-    params.attachmentIds = attachmentIds;
+export function linkUploadedAttachmentToEntity(entityId, entityType, attachmentId) {
+  if (!attachmentId) {
+    throw new Error('Attachment Id can\'t be empty');
   }
-  params = $.param(params, true);
 
-  return fetch(`${eXo.env.portal.context}/${eXo.env.portal.rest}/attachments/${entityType}/${entityId}?${params}`, {
+  return fetch(`${eXo.env.portal.context}/${eXo.env.portal.rest}/attachments/${entityType}/${entityId}`, {
     credentials: 'include',
     method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: attachmentId,
```

Comment:
if it's only about one attachmentId, it would be better to add it in Path param in this case